src/main/java/com/airbnb/billow/AWSDatabase.java

```diff
@@ -253,11 +253,16 @@ public class AWSDatabase {
                 String lastModifiedTimestamp = map.get(SQSQueue.ATTR_LAST_MODIFIED_TIMESTAMP);
                 String queueArn = map.get(SQSQueue.ATTR_QUEUE_ARN);
 
+                com.amazonaws.services.sqs.model.ListQueueTagsRequest tagsRequest =
+                  new com.amazonaws.services.sqs.model.ListQueueTagsRequest().withQueueUrl(url);
+                com.amazonaws.services.sqs.model.ListQueueTagsResult tagsResult =
+                  client.listQueueTags(tagsRequest);
+
```

Comment:
What happens if these requests get rate limited/error out? Would they retry/skip continue..

Reply:
That's a great question. I don't see how that's handled anywhere in billow at all.

Reply:
I would suppose we'd see the failure count tick up (we should have an alert on that), and then set the refreshRate in `reference.conf` from 5m to something higher.